docs/src/development/variables.md

```diff
@@ -158,12 +158,13 @@ Dedicated instances also have the following variables available:
 | Variable name    | Build | Runtime | Description |
```

Comment:
Hmm I can't propose changes on all these lines directly.  Same as above, the variables names miss the styling.

Reply:
GitHub has a limit on unchanged lines for PR comments. That's why you can't comment on them.

Reply:
https://github.com/github/roadmap/issues/347